articles/quickstart/native/ios-swift/_includes/_login_centralized.md

```diff
@@ -0,0 +1,56 @@
+The first step in adding authentication to your iOS application is to provide a way for your users to log in. The fastest, most secure, and most feature-rich way to do this with Auth0 is to use the [centralized login page](https://auth0.com/docs/hosted-pages/login).
```

Comment:
Can we add something to "wrap up" this step of the quickstart? For example, show an image of the hosted Lock in a phone frame. There's a phone frame in the styleguide that can be used, just need to apply the class to the image:

```
<div class="phone-mockup">
  <img src="..." alt="Mobile example screenshot" />
</div>
```